### PR TITLE
Update Mosaic endpoint input/output api

### PR DESCRIPTION
**Pull request by margaretqian**
As noted in prior PRs (https://github.com/hwchase17/langchain/pull/6060, https://github.com/hwchase17/langchain/pull/7348), the input/output format has changed a few times as we've stabilized our inference API. This PR updates the API to the latest stable version as indicated in our docs: https://docs.mosaicml.com/en/latest/inference.html

The input format looks like this:

`{"inputs": [<prompt>]}
`

The output format looks like this:
`
{"outputs": [<output_text>]}
`

LLM Test:

Screenshot 2023-08-16 at 6 20 12 PM

Embedding Test:

Screenshot 2023-08-16 at 6 22 22 PM

### CODE REVIEW

**Comment by baskaryan** [comment]
@margaretqian just let me know when it's ready for review

**Comment by margaretqian** [comment]
@dakinggg could you take a look from the mosaic side?

also @baskaryan apologies for the delay, but this is ready for review now!

**Review by dakinggg — COMMENTED** [review]
lgtm

File: libs/langchain/langchain/llms/mosaicml.py [hunk 1]
```diff
@@ -173,24 +173,9 @@ def _call(
...
-                    else:
-                        raise ValueError(
-                            f"No key data or output in response: {parsed_response}"
-                        )
-                else:
-                    raise ValueError(f"Unexpected response format: {parsed_response}")
             else:
                 raise ValueError(f"Unexpected response type: {parsed_response}")
 
-            text = text[len(prompt) :]
```
[the remark this answers is not in the document]
Comment (margaretqian) [reply libs/langchain/langchain/llms/mosaicml.py:192]: done!

**Comment by dakinggg** [comment]
@hwchase17 @eyurtsev mind taking a look?

**Comment by baskaryan** [comment]
thank you @margaretqian!

**Pull request merged**